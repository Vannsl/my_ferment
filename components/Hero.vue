<template>
  <div>
    <hero-content />
    <div class="relative -mt-12 lg:-mt-24">
      <hero-wave-svg />
    </div>
  </div>
</template>

<script>
import HeroContent from '@/components/HeroContent'
import HeroWaveSvg from '@/components/svg/HeroWave'

export default {
  name: 'Hero',
  components: {
    'hero-content': HeroContent,
    'hero-wave-svg': HeroWaveSvg
  }
}
</script>
