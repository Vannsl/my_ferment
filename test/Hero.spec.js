import { shallowMount } from '@vue/test-utils'
import Hero from '@/components/Hero'

const factory = () => {
  return shallowMount(Hero)
}

describe('Hero', () => {
  it('is a Vue instance', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
