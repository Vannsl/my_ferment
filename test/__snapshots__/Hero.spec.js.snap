// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders properly 1`] = `"<div><hero-content-stub></hero-content-stub> <div class=\\"relative -mt-12 lg:-mt-24\\"><hero-wave-svg-stub></hero-wave-svg-stub></div></div>"`;
